lightkurve/lightcurve.py

```diff
@@ -724,6 +843,43 @@ def to_pandas(self, columns=['time', 'flux', 'flux_err', 'quality',
         """
         return super(KeplerLightCurve, self).to_pandas(columns=columns)
 
+    def to_fits(self, path=None, overwrite=False, **kwargs):
+        """Export the KeplerLightCurve as an astropy.io.fits object.
+
+        Parameters
+        ----------
+        path : string, default None
+            File path, if None returns an astropy.io.fits object.
+        overwrite : bool
+            Whether or not to overwrite the file
+        **kwargs : dict
```

Comment:
This is nitpick: usually we just use ``kwargs`` instead of ``**kwargs`` in the docstrings.

Reply:
I'm fixing this and changing it from `kwargs` to `extra_data`, which is what it is.